### DISCUSSION TITLE
Modify HTML root font size

### DISCUSSION THREAD

**Discussion by johnsicili** [webstudio-is/webstudio-community#53] [Wishlist]
REMs are an awesome unit to use for font-sizes and other properties like padding and margin. But without being able to modify the root HTML font-size they become less useful – especially for responsive design.

I'd love to be able to modify the font-size for the HTML root for each breakpoint. That way, I change that, and all my sizing will scale down as I reduce the width of the screen.

This is one of those things that Webflow has not implemented that I think could be a good win for Webstudio.

Do others find modifying the root size to be good practice?

**Comment by kof** [comment]
We would either have to add it to page settings or actually add html node in the navigator @taylornowotny

**Reply by kof** [reply]
Actually no, it can be part of the tag style source

**Comment by ghost** [comment]
It would be good to be able to set a default unit at both the account level and project level via account & project settings, which we don't have yet.

**Reply by kof** [reply]
I would say having a default unit is a separate discussion

**Reply by ghost** [reply]
Right, misunderstood. Yes this should be part of the style source tags feature. On the design side the UI for this is ready, but we haven't yet determined the ideal process for managing tags through the Style Sources section.

**Comment by johnsicili** [comment]
Launched with the Global Root! https://docs.webstudio.is/university/foundations/anatomy-of-the-webstudio-builder#global-root
